### PR TITLE
Update record-v3.0.0.json

### PR DESCRIPTION
**Pull request by wgresshoff**
This is a fix for Issue #620 (replaced "ref" by "$ref" in "additional_titles" and "additional_descriptions".

### CODE REVIEW

**Comment by wgresshoff** [comment]
Actually I marked the invalicd languages tests in test_additional_description.py and test_additional_title.py as skipped as I don't know how to fix the test.
I hope this is ok, as I didn't just fix the tests but also the datacite serializer to get this done.

**Comment by fenekku** [comment]
I will be squashing :)

File: invenio_rdm_records/records/jsonschemas/records/record-v3.0.0.json [hunk 1]
```diff
@@ -216,7 +216,7 @@
                   "type": "string"
                 },
                 "type": {"$ref": "local://records/definitions-v1.1.0.json#/descriptionType"},
-                "lang": {"ref": "local://records/definitions-v1.1.0.json#/language"}
```
Comment (fenekku) [inline invenio_rdm_records/records/jsonschemas/records/record-v3.0.0.json:219]: :100:

File: tests/services/schemas/test_additional_description.py [hunk 2]
```diff
@@ -89,6 +99,7 @@ def test_invalid_description_type(vocabulary_clear):
     )
 
 
+@pytest.mark.skip(reason="currently don't know how to test this")
```
Comment (fenekku) [inline tests/services/schemas/test_additional_description.py:102]: I will do another pass and test this: https://github.com/inveniosoftware/invenio-rdm-records/issues/624

**Review by fenekku — APPROVED** [review]
Thanks for this @wgresshoff . I suspect there might be more things here, but we can deal with it after.

**Pull request merged**